Xamarin.Forms.Xaml/MarkupExtensions/TemplateBindingExtension.cs

```diff
@@ -24,7 +24,15 @@ public TemplateBindingExtension()
 
 		BindingBase IMarkupExtension<BindingBase>.ProvideValue(IServiceProvider serviceProvider)
 		{
-			return new TemplateBinding(Path, Mode, Converter, ConverterParameter, StringFormat);
+			return new Binding
```

Comment:
shouldn't we obsolete TemplateBindingExtension ?

Reply:
> shouldn't we obsolete TemplateBindingExtension ?
@StephaneDelcroix we're obsoleting `TemplateBinding` in Core for use in code, but still want to keep the `Property="{TemplateBinding Path}"` syntax in XAML don't we?

Reply:
if you think so, I'm fine